packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -364,5 +366,21 @@ export const VtexCommerce = (
         { storeCookies }
       )
     },
+    addresses: async (userId: string): Promise<ProfileAddress[]> => {
```

Comment:
Since this was inside the commerce folder in the index.ts file. I think it could be called `profileAddress`. What do you think fanny?

Reply:
better! Lemme change it! 👍

Reply:
IMO it should be in:

`profile: {
    address: () => {
        ...
    }
`

following the `checkout` and `catalog` structure in the same file

Reply:
since we going to continue to use the previously pattern, I think makes more sense to use

`profile: { addresses: () => {} `